Sources/performance-worker/WorkerService.swift

```diff
@@ -37,12 +38,14 @@ final class WorkerService: Grpc_Testing_WorkerService.ServiceProtocol, Sendable
 
     init() {}
 
-    init(role: Role) {
+    init(role: Role) async throws {
       self.role = role
+      self.initialStats = try await ServerStats()
     }
 
-    init(server: GRPCServer) {
+    init(server: GRPCServer) async throws {
       self.role = .server(server)
+      self.initialStats = try await ServerStats()
```

Comment:
I think the initial stats should be created when the server is created, not when the service is created (because the service might never run a server).